app/2.0/docs/tools/create-element-polymer-cli.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Create an element project with the Polymer CLI
+---
+
+<!-- toc -->
+
+This section shows you how to start an element project.
+
+1.  Create a directory for your element project. It's best practice for the name
```

Comment:
Not sure if the "It's best practice" line is necessary / overall directory naming strategies are probably out of scope for this doc. 

If you do keep it, then explain why its a best practice.